### PR TITLE
change covid19 domain to chicaagolandcovidcommons.org

### PR DESCRIPTION
**Pull request by diw2**
⚠️ DO NOT MERGE ⚠️
when we are ready to switch. We will follow this process:
- [x] create a new dns record to existing elb
- [x] create google client to add new domain as valid redirect 
- [x] update fence-config 
- [x] merge pr and roll out services

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# covid19.datacommons.io/manifest.json
## :warning: Services on branch
- portal

File: covid19.datacommons.io/dashboard/Public/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 <body>
   <div id='header'>
     <div class='header__img-wrapper'>
-      <a href="https://covid19.datacommons.io" target="_blank"><img src="./sponsors-logo/chicagoland_covid19_commons.png" height="100px" /></a>
+      <a href="https://chicaagolandcovidcommons.org" target="_blank"><img src="./sponsors-logo/chicagoland_covid19_commons.png" height="100px" /></a>
```
Comment (paulineribeyre) [inline covid19.datacommons.io/dashboard/Public/index.html:17]: chicaago or chicago?

**Comment by PlanXCyborg** [comment]
# covid19.datacommons.io/manifest.json
## :warning: Services on branch
- portal

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

**Pull request merged**